### ISSUE TITLE
EPIC: Prepare the Join Us page for Homepage Launch

### ISSUE THREAD

**Issue by priyanka02art** [hackforla/website#7672]
### Overview
We need to track existing issues and make new issues to deliver Join Us page complete enough for the Homepage launch

#### Details
We need to track and support the final wrap up items for the homepage launch, so that it can be completed.

### Action Items
- [x] List all the issues related to the label
- [x] Identify what open issues have to be resolved before launch, by classifying the issues into the following categories using MoSCoW method in a comment.
   - Must Have
   - Should Have
   - Could Have
   - Won't Have this time
- [x] Add the following details under each issue
   ```
   - why: [ADD WHY ITS IMPORTANT TO TO THE LAUNCH]
   - status: [ADD STATUS e.g., Icebox, Emergent Requests, New Issue Approval, ERs and epics that are ready to be turned into issues, Prioritized Backlog, In Progress (actively working), Questions / In Review]
      - draft: [true OR false]
      - responsible: [ADD GITHUB HANDLES WITHOUT @]
   - role: [ADD ROLE e.g. data scientist, infrastructure, design, design lead, user research, Research Lead, dev leads, data analyst, BA, hfla leadership, back end/devOps, writing, front end, merge team, technical writer]
   ```
- [x] Once you receive sign-off from Staff PM add to `## Relevance of the issues to the Homepage launch`
- [x] Review comments with `Notes carried forward from epic` if it exists, and make sure all previous items are addressed in the top of the issue.
   - If it specifies new issues to make then add explicit action items for making those issue in a new comment on this issue
      - get sign-off on those new issues to make by Staff PM 
- [x] Identify items that need issues opened by looking at the page and features and seeing what is not already addressed by the existing issues or new issues to make list.
- [x] Make the issues specified
- [x] Link them as sub issues to this issue
- [x] Classify them using MoSCoW and add the details (why, status, draft, responsible, role)
- [ ] Get Sign-off and prioritization of the issues by Staff PM
- [ ] If this page has content that must be checked for accuracy before homepage launch, add the detail to the epic for tracking that (resource 1.05)

### Resources/Instructions
- Parent Epic: https://github.com/hackforla/website/issues/2481

#### Used for creating this issue
- 1.01 Label name: P-Feature: Join Page
    - 1.01.01 Issue Tab: [P-Feature: Join Page](https://github.com/hackforla/website/issues?q=is%3Aopen+label%3A%22P-Feature%3A+Join+Page%22)
    - 1.01.02 Project Board: [P-Feature: Join Page](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=+label%3A%22P-Feature%3A+Join+Page%22)
- 1.02 URL of page on Hack for LA website: https://www.hackforla.org/join
- 1.03 URL of link to code: [repo:hackforla/website NOT language:SVG NOT path:/^_data\/internal\/credits\// Join us](https://github.com/search?q=repo%3Ahackforla%2Fwebsite+NOT+language%3ASVG+NOT+path%3A%2F%5E_data%5C%2Finternal%5C%2Fcredits%5C%2F%2F+Join+us&type=code)
- 1.04 URL of WIKI page: https://github.com/search?q=repo%3Ahackforla%2Fwebsite+Join+Page&type=wikis
   - 1.04.01 URL of future edits to wiki pages: https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki (check this page to see if there are any other new edits in the queue.)
- 1.05 Issue for adding items to a final checklist: #8417

#### Created by this issue
- 2.01 [EXAMPLE would be a link to an issue related to delivering this page]
---
### Must Have
TBD

### Should Have
TBD

### Could Have
TBD

---

last updated: 2026-04-21
Highest number found: 8513

### Must Have
- [ ] #7672
   - why: It is the epic for this page (this issue)
   - status: In progress (actively working)  
      - draft: False
      - responsible: nadiaspn0503  
   - role: Product 

- [ ] #8513
   - why: Our website needs to use the Oxford comma consistently so that the website is correct and uniform. In this issue we will be changing text on the Join Us page.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: Back end/devOps and front end

- [ ] #8512
   - why: Our website needs to use the Oxford comma consistently so that the website is correct and uniform. In this issue we will be changing text on the Join Us page.
   - status: Ready for Prioritization 
      - draft: False
      - responsible: None 
   - role: Back end/devOps and front end

- [ ] #6833 
   - why: The admin meeting times appear on the events page, but the link does not lead to relevant information. Temporary fix is to direct it to the Join Us page, but a dedicated section for joining the admin team should be added in the future.
   - status: Emergent Requests  
      - draft: True
      - responsible: None 
   - role: Design

### Should Have
- [ ] #7510 
   - why: The Donate page header and the Join Us header are different heights from the navigation bar at the top of the page in mobile and tablet views
   - status: New Issue Approval
      - draft: True
      - responsible: JessicaLucindaCheng 
   - role: Back end/devOps and front end

### Could Have
<details><summary>List of all could have issues</summary>

- #3525   
   - why: Ensure consistent text elements on the Join Us page 
   - status: Prioritized Backlog
      - draft: False
      - responsible: None   
   - role: Design

</details>

### Won't have this time
<details><summary>List of all won't have issues</summary>

- #186   

</details> 

#### Templates
```
- [ ] #
   - why: 
   - status: 
      - draft: 
      - responsible: 
   - role:  
```
```
### Issues list associated with this epic
last updated: 
Highest number found:
```

<details><summary>List of all issue by number</summary>

- #8513
- #8512
- #7672
- #7510
- #6833
- #3525
- #186

</details>

**Comment by priyanka02art** [comment]
- [x] #3026
- [x] [Join us page Issues on the project board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22p-feature%3A+Join+Page%22)
URL: https://www.hackforla.org/join
 https://github.com/hackforla/website/issues/5973

**Comment by priyanka02art** [comment]
# List of all issues
- [ ] #7672
- [ ] #7669
- [ ] #7510 
- [ ] #6833
- [ ] #6140  
- [ ] #3525 
- [ ] #186 

## Relevance of the issues to the Homepage launch
### Must Have
- [ ] #7672
   - why: It is the epic for this page (this issue)
   - status: New Issue Approval   
      - draft: True
      - responsible: @priyanka02art 
   - role: Product 
- [ ] #6833 
   - why: The admin meeting times appear on the events page, but the link does not lead to relevant information. Temporary fix is to direct it to the Join Us page, but a dedicated section for joining the admin team should be added in the future.
   - status: Emergent Request   
      - draft: True
      - responsible: None 
   - role: Design

### Should Have
- [ ] #7669 
   - why: Inconsistent URL and possible navigation errors with future code changes
   - status: Emergent Request
      - draft: False
      - responsible: None
   - role: Front End

### Could Have
- [ ] #6140  
   - why: Civic Tech Structures full name is Civic Tech Structure, Inc. and sometimes it makes sense to say Civic Tech Structure, Inc. 501(c)(3) or even Civic Tech Structure, Inc. a non profit 501(c)(3). Different pages have different listings 
   - status: Emergent Requests
      - draft: False
      - responsible: @ExperimentsInHonesty   
   - role: Product
- [ ] #3525   
   - why: Ensure consistent text elements on the Join Us page 
   - status: Prioritized Backlog
      - draft: False
      - responsible: None   
   - role: Design

### Won't have this time
- [ ] #7510    
   - why: The Donate and Join Us page headers have inconsistent heights from the navigation bar in mobile and tablet views (screen width 767px or less), but UX is unaffected  
   - status: New Issue Approval
      - draft: True
      - responsible: @JessicaLucindaCheng 
   - role: Front End
- [ ] #186     
   - why: Analyze how data from survey forms/stakeholder contact can be leveraged 
   - status: New Issue Approval
      - draft: False
      - responsible: None 
   - role: User Research

#### Template
```
   - [ ] # 
   - why: 
   - status: 
      - draft
      - responsible: 
   - role: 
``` 

```
### Issues list associated with this epic
last updated: 
Highest number found:
```

**Comment by ExperimentsInHonesty** [comment]
@charmainemak09 Please provide update

<details><summary>Instructions</summary>

1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
</details> 

You can use this template
```
1. Progress: 
2. Blockers: 
3. Availability:
4. ETA:
5. Pictures (if necessary): 
```

**Comment by priyanka02art** [comment]
Sent email to Charmaine on 2025-06-05

> Hi Charmaine,
> 
> Please review the issues assigned to you and provide a brief update on each one by asap. Specifically:
> 
> Current status (e.g., in progress, blocked, completed)
> 
> Any dependencies or blockers
> 
> Estimated timeline for completion (if not yet done)
> 
> You can leave your updates directly as comments on the respective issues so everything stays in one place.
> 
> Let me know if you have any questions or need help unblocking anything.
> 
> Thanks,
> Priyanka

**Comment by nadiaspn0503** [comment]
2026-08-18
Using this search 
[state:open label:"P-Feature: Join Page" is:issue ](https://github.com/hackforla/website/issues?q=state%3Aopen%20label%3A%22P-Feature%3A%20Join%20Page%22%20is%3Aissue)

This is the highest issue number found 7672 it is addressed at the top of this issue.
